spec/controllers/groups/posts_controller_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+RSpec.describe Groups::PostsController, type: :controller do
+  let!(:user) { create(:user) }
+  let!(:group) { create(:group) }
+
+  before { sign_in user }
+
+  describe 'Post#Сreate' do
+    context 'when parameters is valid' do
+      let(:valid_parameters) { { body: 'Something' } }
+
+      it 'create new post' do
+        post :create, params: { group_id: group.id, post: valid_parameters }
+        expect(group.posts.last.body).to eq(valid_parameters[:body])
+        expect(response).to have_http_status(:redirect)
+      end
+
+      it 'post was created' do
+        expect do
+          post :create, params: { group_id: group.id, post: valid_parameters }
+        end.to change(Post, :count).by(1)
+      end
```

Comment:
you can merge those two specs into one it will work two times faster

```suggestion
      it 'creates new post' do
        expect do
          post :create, params: { group_id: group.id, post: valid_parameters }
        end.to change(Post, :count).by(1)
        expect(group.posts.last.body).to eq(valid_parameters[:body])
        expect(response).to have_http_status(:redirect)
      end
```